ratis-server/src/test/java/org/apache/ratis/InstallSnapshotFromLeaderTests.java

```diff
@@ -168,12 +169,12 @@ public long takeSnapshot() {
           FileUtils.createDirectories(snapshotRoot);
           FileUtils.createDirectories(file1.getParentFile());
           FileUtils.createDirectories(file2.getParentFile());
-          FileUtils.createNewFile(file1.toPath());
-          FileUtils.createNewFile(file2.toPath());
+          FileUtils.newOutputStream(file1.toPath(), StandardOpenOption.CREATE_NEW);
+          FileUtils.newOutputStream(file2.toPath(), StandardOpenOption.CREATE_NEW);
```

Comment:
Shouldn't these be closed immediately?  (applies to old code, too)

What is the benefit compared to the following?

```
file1.createNewFile();
file2.createNewFile();
```

Reply:
@adoroszlai , the test tries to create a few different snapshot files and test if they can be transferred correctly.  So these files should be closed locally.

Reply:
My point is that the test is leaking these two output streams.  It's not a big deal, since it's just a unit test.

> these files should be closed locally

Can you please clarify that?

Reply:
Oops, sorry, I have not pushed the last commit.